lib/src/analysis_options.dart

```diff
@@ -5,6 +5,8 @@
 const String analysisOptions = '''
 analyzer:
   strong-mode: true
+  errors:
+    uri_has_not_been_generated: ignore
```

Comment:
No!

I think the idea is: if the source package has this setup – then copy it.

Just whitelist `uri_has_not_been_generated` for pass-through